pandas/tests/io/test_html.py

```diff
@@ -406,9 +406,7 @@ def test_invalid_table_attrs(self, banklist_data):
             )
 
     def _bank_data(self, path, *args, **kwargs):
```

Comment:
Remove the *args here as well?

```suggestion
    def _bank_data(self, path, **kwargs):
```